spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/optimizer/RewriteDelete.scala

```diff
@@ -34,12 +34,11 @@ import org.apache.spark.sql.catalyst.plans.logical.DeleteFromTable
 import org.apache.spark.sql.catalyst.plans.logical.Filter
 import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
 import org.apache.spark.sql.catalyst.plans.logical.Project
-import org.apache.spark.sql.catalyst.plans.logical.RepartitionByExpression
 import org.apache.spark.sql.catalyst.plans.logical.ReplaceData
 import org.apache.spark.sql.catalyst.plans.logical.Sort
 import org.apache.spark.sql.catalyst.rules.Rule
 import org.apache.spark.sql.catalyst.utils.DistributionAndOrderingUtils
-import org.apache.spark.sql.catalyst.utils.PlanUtils.isIcebergRelation
+import org.apache.spark.sql.catalyst.utils.PlanUtils._
```

Comment:
This should cause style checks to fail because we don't use wildcard imports.

Reply:
I didn't realize that. Let me fix that.

Reply:
Is this only at the top level?
Is this why in this class, there is
```
import org.apache.spark.sql.catalyst.utils.DistributionAndOrderingUtils
```
and then
```
  import DistributionAndOrderingUtils._
```
inside the class?

Reply:
We should talk about locally scoped imports. We definitely want to enforce this at a top level.